packages/workflow/src/Expression.ts

```diff
@@ -132,7 +157,20 @@ export class Expression {
 	 * @returns {(NodeParameterValue | INodeParameters | NodeParameterValue[] | INodeParameters[] | undefined)}
 	 * @memberof Workflow
 	 */
-	getComplexParameterValue(node: INode, parameterValue: NodeParameterValue | INodeParameters | NodeParameterValue[] | INodeParameters[], mode: WorkflowExecuteMode, defaultValue: NodeParameterValue | INodeParameters | NodeParameterValue[] | INodeParameters[] | undefined = undefined, selfData = {}): NodeParameterValue | INodeParameters | NodeParameterValue[] | INodeParameters[] | undefined {
+	getComplexParameterValue(
+		node: INode,
+		parameterValue: NodeParameterValue | INodeParameters | NodeParameterValue[] | INodeParameters[],
+		mode: WorkflowExecuteMode,
+		/* eslint-disable @typescript-eslint/indent */
+		defaultValue:
+			| NodeParameterValue
+			| INodeParameters
+			| NodeParameterValue[]
+			| INodeParameters[]
+			| undefined = undefined,
```

Comment:
One of the rare cases where prettier conflicts with eslint, decided to go with the prettier solution, because it looks cleaner.

eslint would put the `|` on the same indentation as `defaultValue:`

Reply:
this probably references to this comment (https://github.com/n8n-io/n8n/pull/1566#discussion_r598842016), the disabling can probably removed when the rule is removed 👍🏼